tool/tbot/main.go

```diff
@@ -387,13 +544,18 @@ func renewLoop(ctx context.Context, cfg *config.BotConfig, client auth.ClientI,
 	ticker := time.NewTicker(cfg.RenewInterval)
 	defer ticker.Stop()
 	for {
+		// Make sure we can still write to the bot's destination.
```

Comment:
I had to scroll down a long way to see the ticker read. Maybe it would be worth extracting the body of this loop to a separate func, so it's easy to see that renew loop is just orchestrating some function on each tick.

Reply:
I'll pull that into its own function.